frontend/src/components/PopOverComponent.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Box, Typography, Button, Popover } from '@mui/material';
+
+const PopoverComponent = ({ open, handleClose, selectedPlaceProperties, anchorElRef }) => {
+  return (
+    <div ref={anchorElRef} style={{ display: 'none', position: 'absolute' }}>
+      <Popover
+        id="popover"
+        open={open}
+        onClose={handleClose}
+        anchorEl={anchorElRef.current}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'center',
+        }}
+      >
+        <Box>
+          <Typography variant="h6" component="h2">
+            Location Properties
+          </Typography>
+          <Typography variant="body1" component="div" sx={{ mt: 2 }}>
+            <div>
```

Comment:
Given your use of material-ui, it's preferable to employ controlled components like Box, rather than relying solely on the vanilla div element